_topic_map.yml

```diff
@@ -541,3 +541,12 @@ Topics:
     File: prometheus-alertmanager-and-grafana
 - Name: Application monitoring
   File: application-monitoring
+---
+Name: Container-native Virtualization
+Dir: cnv
+Distros: openshift-enterprise
+Topics:
+- Name: Attaching a virtual machine to multiple networks
+  File: cnv-attaching-vm-multiple-networks
+- Name: Configuring PXE booting for virtual machines
+  File: cnv-configuring-pxe-booting
```

Comment:
small nit here please add a newline

Reply:
Done in https://github.com/openshift/openshift-docs/pull/14351/commits/53f3f7e09d33683882862d62e7d7d9fe55f7644e